src/entities/subject/subject.service.ts

```diff
@@ -1,6 +1,7 @@
 import { Injectable, Inject } from '@nestjs/common'
 import { CONTEXT } from '@nestjs/graphql'
 import { InjectModel } from '@nestjs/mongoose'
+import mongoose from 'mongoose'
```

Comment:
@frankmangone you have mongoooooose below already :)

Reply:
Lol I'm actually not using it. Removed that import.